src/Microsoft.Crank.AzureDevOpsWorker/Program.cs

```diff
@@ -133,7 +134,10 @@ private static async Task MessageHandler(ProcessMessageEventArgs args)
 
                 if (Verbose)
                 {
-                    Console.WriteLine($"{LogNow} Payload (Base64): {Convert.ToBase64String(bodyArray)}");
+                    // Truncate verbose payload logging to avoid huge console output
+                    var payloadB64 = Convert.ToBase64String(bodyArray);
+                    var max = 2048;
```

Comment:
Is this truncation something we want? I think it makes sense to keep logging size from growing too big.

Reply:
You can display the content size instead of the content

Reply:
Why resolved?

Reply:
Updated the log to only log the size of the payload.